modules/virt-configuring-node-exporter-service.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-node-exporter-service_{context}"]
+= Configuring the node exporter service
+
+The `node-exporter` agent is deployed on every node and collects metrics about every node in a cluster. Configure the `node-exporter` agent as a service to expose metrics from virtual machines.
```

Comment:
the node-exporter agent is deployed on every virtual machine in the cluster that you wish to collect metrics that are internal to the virtual machine and its processes.

Reply:
Updated the content. Adjusted both sentences in this paragraph.